app/views/layouts/partials/_footer.html.slim

```diff
@@ -48,15 +48,31 @@ footer.footer
           a.footer__social__link href=url
             i.icon class="icon--#{type}"
         
+      / - if Rails.configuration.published_locales.length > 1
+      /   .footer__language
+      /     .footer__header
+      /       = translate('footer.languages')
+      /       a.icon.icon--language
+      /     - Rails.configuration.published_locales.each do |locale|
+      /       a.footer__language__link href=root_url(host: Rails.configuration.locale_hosts[locale])
+      /         = locale.to_s.upcase
+      /       - if locale != Rails.configuration.published_locales.last
+      /         .footer__language__separator  |
+
       - if Rails.configuration.published_locales.length > 1
-        .footer__language
-          .footer__header
-            = translate('footer.languages')
-            .icon.icon--language
-          - Rails.configuration.published_locales.each do |locale|
-            a.footer__language__link href=root_url(host: Rails.configuration.locale_hosts[locale])
-              = locale.to_s.upcase
-            - if locale != Rails.configuration.published_locales.last
-              .footer__language__separator  | 
+        .footer__language id="language"
```

Comment:
Can you change this id to `language-switcher` just so that it's a little bit more descriptive, and matches the JS file.